### DISCUSSION TITLE
sh1mmer error when trying to boot to SD card in recovery screen

### DISCUSSION THREAD

**Discussion by grandpa1946** [3kh0/ext-remover#502] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i downloaded the bin file, then i went to the [wax4web](https://sh1mmer.me/builder.html) page and clicked the start button. I used the patch bin file with the chrome recovery utility, and when i did every step right (after going into dev mode etc.) it stops with a red error message. i will write the error message here later, i forgot what it said...

**Comment by 3kh0** [comment, marked as answer]
We can not help if you dont tell us the error

**Reply by grandpa1946** [reply]
i already got the answer, i had to use a usb flash drive, not a SD card.
